### PR TITLE
Simplify some stuff

### PR DESCRIPTION
**Pull request by Sword352**
ò

### CODE REVIEW

File: source/display/objects/game/Note.hx [hunk 1]
```diff
@@ -80,20 +80,13 @@ class Note extends FlxSprite
...
-			case 2:
-				x += swagWidth * 2;
-				animation.play('greenScroll');
-			case 3:
-				x += swagWidth * 3;
-				animation.play('redScroll');
+			case 0: animation.play('purpleScroll');
+			case 1: animation.play('blueScroll');
+			case 2: animation.play('greenScroll');
+			case 3: animation.play('redScroll');
```
Comment (libauror-a) [inline source/display/objects/game/Note.hx:89]: [this code has already been refactored in my local changes](https://github.com/ThePlank/PlankEngine/assets/75984453/e85329f9-8fd2-4460-8ac5-4dd5ab561759)
also thank you for reminding me to actually commit my changes

File: source/states/game/PlayState.hx [hunk 2]
```diff
@@ -1651,14 +1651,12 @@ class PlayState extends states.abstr.MusicBeatState
...
-		FlxTween.cancelTweensOf(iconP1);
-		FlxTween.cancelTweensOf(iconP2);
-		FlxTween.tween(iconP1, {"scale.x": 1, "scale.y": 1}, Conductor.crochet / 1000, {ease: FlxEase.backOut});
-		FlxTween.tween(iconP2, {"scale.x": 1, "scale.y": 1}, Conductor.crochet / 1000, {ease: FlxEase.backOut});
+		for (icon in [iconP1, iconP2]) {
+			icon.scale.set(1.3, 0.75);
+			FlxTween.cancelTweensOf(icon);
+			FlxTween.tween(icon, {"scale.x": 1, "scale.y": 1}, Conductor.crochet / 1000, {ease: FlxEase.backOut});
+		}
+
```
Comment (libauror-a) [inline source/states/game/PlayState.hx:1659]: and i changed the icon bop to be more inline with most engines so ą

Comment (Sword352) [reply source/states/game/PlayState.hx:1659]: Should i close this pr idfk

File: source/display/objects/game/Note.hx (hunk 1 above)
Comment (Sword352) [reply source/display/objects/game/Note.hx:89]: nice font

**Pull request closed without merging**